pilot/pkg/features/pilot.go

```diff
@@ -206,7 +206,7 @@ var (
 
 	DistributionHistoryRetention = env.RegisterDurationVar(
 		"PILOT_DISTRIBUTION_HISTORY_RETENTION",
-		time.Minute*1,
+		time.Minute*0,
```

Comment:
what is the motivation for this change?

Reply:
This was not supposed to make the final CL.. removed.